packages/fuels-accounts/src/kms/aws/client.rs

```diff
@@ -0,0 +1,63 @@
+pub use aws_config::{
+    default_provider::credentials::DefaultCredentialsChain, BehaviorVersion, Region, SdkConfig,
+};
+pub use aws_sdk_kms::config::Credentials;
+use aws_sdk_kms::Client;
+
+#[derive(Clone)]
+pub struct AwsConfig {
+    sdk_config: SdkConfig,
+}
+
+impl AwsConfig {
+    pub async fn from_environment() -> Self {
+        let loader = aws_config::defaults(BehaviorVersion::latest())
+            .credentials_provider(DefaultCredentialsChain::builder().build().await);
+
+        Self {
+            sdk_config: loader.load().await,
+        }
+    }
+
```

Comment:
Since we're now `pub use *` ing stuff from the aws crates, I'd drop there wrappers. 

Use the types from the aws crates themselves.